docs/content/preview/contribute/core-database/includes/java.md

```diff
@@ -4,13 +4,20 @@ private = true
 +++
 -->
 
-YugabyteDB core is written in C and C++, but the repository contains Java code needed to run some tests and sample applications. To build the Java part, you need:
+YugabyteDB core is written in C and C++, but the repository contains Java code needed to run some tests and sample applications.
+To build the Java part, you need:
 
-* Java Development Kit (JDK) 1.8. JDK installers for Linux and macOS can be downloaded from [OpenJDK](http://jdk.java.net/), [AdoptOpenJDK](https://adoptopenjdk.net/), or [Azul Systems](https://www.azul.com/downloads/zulu-community/). Homebrew users on macOS can install using `brew install openjdk`.
+* Java Development Kit (JDK) 8 or 11.
+  JDK can be downloaded from [OpenJDK](http://jdk.java.net/archive), [AdoptOpenJDK](https://adoptopenjdk.net/), or [Azul Systems](https://www.azul.com/downloads/zulu-community/).
```

Comment:
Since 8 is no longer actively maintained (security updates only), we should use 11 as the example.
```suggestion
  JDK can be downloaded from [OpenJDK](http://jdk.java.net/archive), [AdoptOpenJDK](https://adoptopenjdk.net/), or [Azul Systems](https://www.azul.com/downloads/zulu-community/). Homebrew users on macOS can install using `brew install openjdk@11`.
```